keel-sql/src/main/resources/jooqConfig.xml

```diff
@@ -14,6 +14,24 @@
         DATABASECHANGELOG
         | DATABASECHANGELOGLOCK
       </excludes>
+      <forcedTypes>
+        <!-- Automatically convert JSON columns to String -->
+        <forcedType>
+          <name>LONGVARCHAR</name>
+          <includeTypes>JSON</includeTypes>
+        </forcedType>
+        <!-- Automatically convert TIMESTAMP to Instant -->
+<!--        <forcedType>-->
```

Comment:
It turns out JOOQ has built-in support for converting between time types. This causes the generator to declare the type of the field as `org.jooq.impl.SQLDataType.INSTANT` instead of org.jooq.impl.SQLDataType.LOCALDATETIME`, and sounded like it would be exactly what we needed, except inserts fail with errors like these:
```
org.jooq.exception.DataAccessException: SQL [insert into `keel`.`delivery_config_last_checked` (`delivery_config_uid`, `at`) values (?, ?) on duplicate key update `keel`.`delivery_config_last_checked`.`at` = ?]; Data truncation: Incorrect datetime value: '2020-03-13 21:24:06.244+00:00' for column 'at' at row 1

	at org.jooq_3.12.3.MYSQL.debug(Unknown Source)
	at org.jooq.impl.Tools.translate(Tools.java:2717)
	at org.jooq.impl.DefaultExecuteContext.sqlException(DefaultExecuteContext.java:755)
	at org.jooq.impl.AbstractQuery.execute(AbstractQuery.java:382)
        ...
Caused by: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Incorrect datetime value: '2020-03-13 21:24:06.244+00:00' for column 'at' at row 1
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:104)
        ...
```
😢